docs/blog/content/material/database.en.md

```diff
@@ -4,4 +4,296 @@ weight = 6
+
+A two-phase transaction that lacks concurrency guarantee cannot be called a perfect distributed transaction solution; a flexible transaction that lacks the original support of ACID cannot even be called a database transaction, which is more suitable for service layer transaction processing.
+
+At present, it is difficult to find a distributed transaction solution that can be used universally without trade-offs.
+
+A new generation of distributed transaction middleware JDTX
+
+JDTX is a self-developed distributed transaction middleware by JD.com, which has not yet been open sourced. Its design goals are strongly consistent (supporting ACID's transaction meaning), high performance (not less than local transaction performance), 1PC (completely abandoning two-phase commit and two-phase lock) fully distributed transaction middleware, currently available for Relational Database. It adopts a completely open SPI design method to provide the possibility of interfacing with NoSQL, and can maintain multiple heterogeneous data in the same transaction in the future.
+
+JDTX uses a fully self-developed transaction processing engine to convert data in SQL operations into KV (key-value pairs), and on the basis of it, implements the MVCC (multi-version snapshot) transaction visibility engine and the database design concept. Similar WAL (Write-ahead Logging System) storage engine. You can understand the composition of JDTX through the following architecture diagram:
+
+![](https://shardingsphere.apache.org/blog/img/database9.jpg)
+
+Its design feature is to separate the data in the transaction (called active data) from the data that is not in the transaction (called placement data). After the active data is placed on the WAL, it is saved in the form of KV to the MVCC memory engine. Placed data is synchronized to the final storage medium (such as a relational database) by asynchronously flashing the REDO logs in WAL in a flow controllable manner. The transactional memory query engine is responsible for retrieving relevant data from the active data in KV format using SQL, merging it with the data on the market, and obtaining the data version that is visible to the current transaction according to the transaction isolation level.
+
+JDTX reinterprets the database transaction model with a new architecture. The main highlights are:
+
+**1. The internalized distributed transaction is a local transaction**
+
+JDTX's MVCC engine is a centralized cache. It can internalize the two-phase commit to the one-phase commit to maintain the atomicity and consistency of the data in a single node, that is, reduce the scope of distributed transactions to the scope of local transactions. JDTX guarantees the atomicity and consistency of transaction data by ensuring that all access to transaction data passes through the active data of the MVCC engine + the final data-end data combination.
+
+**2. Non-destructive transaction isolation mechanism**
```

Comment:
It is better to change `Non-destructive transaction isolation mechanism` to `Fully support all transaction isolation levels`